src/commands/init.ts

```diff
@@ -263,4 +219,15 @@ export default class Init extends Command {
     this.log('| Use `sap-cloud-sdk add-cx-server` to create the setup script  |');
     this.log('+---------------------------------------------------------------+');
   }
+
+  private printSuccessMessageScaffold() {
+    this.log('+---------------------------------------------------------------+');
+    this.log('| ✅ Init finished successfully.                                |');
+    this.log('|                                                               |');
+    this.log('| 🚀 Next step: Deploy your application (`npm run deploy`)      |');
+    this.log('|                                                               |');
+    this.log('| 🔨 Consider setting up Jenkins to continuously build your app |');
```

Comment:
Maybe this is overkill, but you could specify duplicate logs only once and invoke them from here. Maybe even in a  separate module, e. g.:

```ts
// message printer

function border(): string[] {
  return ['+---------------------------------------------------------------+'];
}

function emptyLine(): string {
  return ['|                                                               |'];
}

function finishedSucessFully(command: string): string {
  return [`| ✅${command.toUpperCase()} finished successfully.                                |`];
}

function considerCXServer(): string[] {
  return [
    '| 🔨 Consider setting up Jenkins to continuously build your app |',
    '| Use `sap-cloud-sdk add-cx-server` to create the setup script  |'
  ];
}
```

You could even go further and make sure all messages have the correct whitespaces and so on... but as I said, maaaaaybe this is a bit too much.

Reply:
I thought about it and decided it would be easier to understand and edit in the current state. Let me know if you disagree.

Reply:
It's intentional. We have very different assumptions in the 2 scenarios which allows us to tell the user how to start locally and make deploying a single step instead of a 4 step guide.